scripts/create_topological_sort_of_all_services.py

```diff
@@ -106,9 +106,15 @@ def make_graph() -> Tuple[Dict[str, List[str]], Set[str]]:
                     parsed_nodes = parsed_script.body
                     for parsed_node in parsed_nodes:
                         # For require statements.
-                        if parsed_node.type == 'ExpressionStatement' and (
-                                parsed_node.expression.callee.name == (
-                                    'require')):
+                        # The dep_lines contains only the expressions beginning
+                        # with 'require' keyword. This is made sure by the code
+                        # section above. Hence, a branch where the below if
+                        # condition is false will never arise and hence, we add
+                        # the 'no branch' flag.
```

Comment:
Given this, why not just remove the `if` condition? Maybe try removing the `if` statement and check that the same topological sort is generated

Reply:
Removed the `if` condition and all the checks are passing.

Reply:
Great! Do you get the same output from running `create_topological_sort_of_all_services.py`? (just in case the tests aren't strict enough)

Reply:
Yeah, the sorted outputs are the same after and before the changes.